src/dist/replication/lib/replica_stub.cpp

```diff
@@ -85,6 +85,12 @@ replica_stub::replica_stub(replica_state_subscriber subscriber /*= nullptr*/,
     _log = nullptr;
     _primary_address_str[0] = '\0';
     install_perf_counters();
+
+    _max_allowed_write_size = dsn_config_get_value_uint64(
+        "pegasus.server",
+        "max_allowed_write_size",
+        1000000,
```

Comment:
I think use 1e6 is better. Because use 1000000, we must count the number of '0' to get the value.

Reply:
`1<<20` would be better, it represent 1M accurately